### PR TITLE
Upgrade webpack and react dependencies

### PR DESCRIPTION
**Pull request by badrey**
These changes fix issues with SSR of emotion library

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -69,15 +69,15 @@
...
+        "react": "16.6.1",
+        "react-dom": "16.6.1",
+        "react-wildcat-prefetch": "8.0.0",
         "resolve": "^1.1.7",
         "rimraf": "^2.5.2",
         "shelljs": "^0.7.0",
         "sinon": "^2.0.0-pre.5",
         "sinon-chai": "^2.8.0",
-        "webpack": "^2.6.1",
+        "webpack": "3.12.0",
```
Comment (miblanchard) [inline package.json]: Will going to webpack 4 break things and/or take a significant more amount of time?

File: example/package.json [hunk 2]
```diff
@@ -9,12 +9,12 @@
         "koa-static": "^2.0.0",
         "prop-types": "^15.5.10",
         "radium": "0.24.0",
-        "react-dom": "16.4.2",
+        "react-dom": "16.6.1",
```
[the remark this answers is not in the document]
Comment (miblanchard) [reply example/package.json]: Or do we need to keep this in sync with that is in the web repo?

[the remark this answers is not in the document]
Comment (badrey) [reply example/package.json]: @miblanchard Yes, I did it on purpose to keep them in sync to be on the safe side

File: package.json (hunk 1 above)
Comment (badrey) [reply package.json]: I upgraded to the same version as on web. It helped with `emotion` issues. So, I guess it is better to keep them in sync

**Pull request merged**